### ISSUE TITLE
copy wiki from COSIMA/ACCESS-OM2?

### ISSUE THREAD

**Issue by aekiss** [ACCESS-NRI/ACCESS-OM2#64]
There's a lot of useful info for users in the COSIMA ACCESS-OM2 wiki https://github.com/COSIMA/access-om2/wiki

Should it be copied here? Or at least linked to?

Most of it is still relevant.

**Comment by aidanheerdegen** [comment]
I agree at the very least it should be linked. Copying would require some review of the material in case there were things that were no longer relevant, and I don't know if there are the resources for that at the moment.
